src/style.css

```
@@ -0,0 +1,94 @@
+@font-face {
+  font-weight: 300;
+  font-family: "Avenir Book";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+}
+@font-face {
+  font-weight: 600;
+  font-family: "Avenir Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-weight: 300;
+  font-family: "Avenir Book", Arial, sans-serif;
+}
+
+body {
+  padding-top: 12px;
+}
+
+.form {
+  position: relative;
+}
+
+.search__bar {
+  width: 100%;
+  height: 100%;
+  font-size: 16px;
+  font-family: inherit;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```

Краще робити відступи між різними групами правил
```suggestion
  width: 100%;
  height: 100%;
  
  font-size: 16px;
  font-family: inherit;
  
  border-radius: 4px;
  border: 1px solid #e1e7ed;
  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```